contracts/DCAHubCompanion/DCAHubCompanionHubProxyHandler.sol

```diff
@@ -89,6 +89,18 @@ abstract contract DCAHubCompanionHubProxyHandler is IDCAHubCompanionHubProxyHand
     _hub.increasePosition(_positionId, _amount, _newSwaps);
   }
 
+  /// @inheritdoc IDCAHubCompanionHubProxyHandler
+  function increasePositionWithAllBalanceProxy(
```

Comment:
I'm thinking name should probably be `increasePOsitionWithAlllCompanionsBalance`? Or something along those lines? Because with `AllBalance` it seems that we are talking about the user's balance.

Reply:
Also: why not re-use the `increasePositionProxy` code?

Reply:
Just to optimize gas. I would need to call `_hub.userPosition(_positionId).from;` twice (once here and once inside `increasePositionProxy`). I though about creating an internal function that did approve + increase, but it really didn't make much sense to refactor everything for two lines of code

Reply:
No, for sure. Refactoring everything doesn't make sense. Just that it adds overhead on testing (since you can't just test the "internal" call. But its ok then.